<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataBaseSettingsWidget</class>
 <widget class="QWidget" name="DataBaseSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>255</width>
    <height>153</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ustawienia bazy danych</string>
  </property>
  <property name="windowIcon">
   <iconset resource="guiIconsResources.qrc">
    <normaloff>:/icons/server-database.png</normaloff>:/icons/server-database.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Adres</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="addressLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Has≈Ço</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="loginLineEdit"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="passLineEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Baza</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="baseLineEdit"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="guiIconsResources.qrc"/>
 </resources>
 <connections/>
</ui>
